### ISSUE TITLE
Ratings not working in Typo3 7.6 since v1.13

### ISSUE THREAD

**Issue by Melana13** [pagemachine/ats#128]
When clicking the tab "Rating" or "Rating (Perso)" in application single view within the application module, the following error message occurs: Required argument "options" was not supplied.
The Error doesn't occur in ATS version 1.12.1.

When "Enable legacy backend TS" is not set, no applications are shown at all in the module. This error occurs in Typo3 8.7 and ATS version 1.12.1 as well. 

Typo3 versions tested: 7.6.27 / 8.7.27
ATS versions tested: 1.12.1, 1.13.0 , 1.14.0, 1.15.0

**Comment by Melana13** [comment]
Thank you for your help.
I didn't tell the module where to pull data from. Now this problem is solved.

But still I get the error message on ratings.
It is a fresh typo3 installation 7.6.27 and I just installed felogin 7.6.27, static_info_tables 6.5.1 and ats 1.15.1 via extension manager.
Everything I did so far was to set the following constants:
plugin.tx_ats.settings.companyData.name = XXX
plugin.tx_ats.settings.applicationPage = 10
plugin.tx_ats.settings.loginPage = 8
plugin.tx_ats.settings.includeJQuery = 0
plugin.tx_ats.persistence.storagePid = 9
plugin.tx_ats.settings.feUserGroup = 1

and in setup:
module.tx_ats.persistence.storagePid = 9
